portality/scripts/reject_update_requst.py

```diff
@@ -0,0 +1,9 @@
+from portality.models import Application
+from portality.constants import APPLICATION_STATUS_REJECTED
+
+UR_TO_REJECT = "5742fbfb237f4b38a7b58d54951727d6"
+
+if __name__ == "__main__":
+    ur = Application.pull(UR_TO_REJECT)
+    ur.set_application_status(APPLICATION_STATUS_REJECTED)
+    ur.save()
```

Comment:
We should make sure to run through all the other checks that happen during application rejection in case they need to be done too.  See https://github.com/DOAJ/doaj/blob/develop/portality/bll/services/application.py#L18

I'd consider at least the following:

* we should add a note to say what was done and why
* check the state of the current_journal value, and set it to related_journal if needed
* check the related journal and see if current_application also needs to be moved to related_application (we can probably just do this manually on the record, rather than adding the code)

Reply:
Sorry, I assigned you too soon. Done.